recipes/qt/6.x.x/conanfile.py

```diff
@@ -788,7 +788,6 @@ def _create_plugin(pluginname, libname, type, requires):
             _create_module("QuickTest", ["Test"])
 
         if self.options.qttools and self.options.gui and self.options.widgets:
-            _create_module("LinguistTools", ["Gui", "Widgets"])
             _create_module("UiPlugin", ["Gui", "Widgets"])
```

Comment:
```suggestion
            self.cpp_info.components["qtLinguistTools"].names["cmake_find_package"] = "LinguistTools"
            self.cpp_info.components["qtLinguistTools"].names["cmake_find_package_multi"] = "LinguistTools"
            _create_module("UiPlugin", ["Gui", "Widgets"])
```
This should allow to create a LinguistTools module, which gives access to qt6_create_translation  and qt6_add_translation cmake macros

Reply:
We should probably do the same for Platform above:
replace
```
        _create_module("Platform")
        self.cpp_info.components["qtPlatform"].libs = [] # this is a collection of abstract classes, so this is header-only
        self.cpp_info.components["qtPlatform"].libdirs = []
```
with
```
            self.cpp_info.components["qtPlatform"].names["cmake_find_package"] = "Platform"
            self.cpp_info.components["qtPlatform"].names["cmake_find_package_multi"] = "Platform"
```